cli/main.c

```diff
@@ -137,7 +137,7 @@ int pax_handler(const char *optarg, void *value_addr,
 
 static int list(int argc, char **argv)
 {
-	struct switchtec_device_info *devices;
+	struct switchtec_device_info *devices = NULL;
```

Comment:
I don't understand how this changes anything... Doesn't switchtec_list() assign to devices even when there are no devices?

Reply:
In switchtec_list() in lib/platform.linux.c devices is not set until after scandir finds no devcies.

Reply:
Ah, right. Thanks!